Src/Graphics/Shader/ShaderOGL3.cpp

```diff
@@ -38,25 +38,24 @@ ShaderOGL3::~ShaderOGL3() {
 void ShaderOGL3::extractVertexUniforms(const String& vertexSource) {
     std::vector<ParsedShaderVar> vertexUniforms;
     extractShaderVars(vertexSource, "uniform", vertexUniforms);
-    std::vector<StructuredData::ElemLayout::Entry> layoutEntries;
-    for (int i = 0; i < (int)vertexUniforms.size(); i++) {
+    int vertexUniformSize = 0;
```

Comment:
I would rename this to `vertexUniformOffset`. The variable itself represents the addition of the sizes of previous uniforms, not the size of any single uniform.

Reply:
Is that all??